api/room/schema_query.py

```diff
@@ -211,3 +232,35 @@ def resolve_bookings_analytics_count(self, info, start_date, end_date):  # noqa:
         analytics = RoomAnalyticsRatios.get_bookings_analytics_count(
             self, query, start_date, end_date)
         return analytics
+
+    def resolve_analytics_for_daily_room_events(self, info,
+                                                start_date, end_date
+                                                ):
+        start_date, end_date = CommonAnalytics().convert_dates(
+            start_date, end_date
+        )
+        query = Room.get_query(info)
+        all_events, all_dates = RoomSchedules().get_all_room_schedules(
```

Comment:
Kindly explain why there is a need to assign two different variables to the same assignment value.

Reply:
<img width="683" alt="image" src="https://user-images.githubusercontent.com/30701246/51996476-e3c4e600-24c5-11e9-96f2-9ae13dd8d212.png">
@Bonifase the above screenshot shows the two values. I hope this clarifies your concern.